operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -171,6 +173,14 @@ func (t Translator) checkInitBundleSecret(ctx context.Context, sc platform.Secur
 	return nil
 }
 
+func (t Translator) globalMonitoring(m *platform.GlobalMonitoring) *translation.ValuesBuilder {
+	enabled := translation.NewValuesBuilder()
+	enabled.SetBoolValue("enabled", m.IsOpenShiftMonitoringEnabled())
+	openshift := translation.NewValuesBuilder()
+	openshift.AddChild("openshift", &enabled)
+	return &openshift
```

Comment:
```suggestion
	openshiftMonitoring := translation.NewValuesBuilder()
	openshiftMonitoring.SetBoolValue("enabled", m.IsOpenShiftMonitoringEnabled())
	globalMonitoring := translation.NewValuesBuilder()
	globalMonitoring.AddChild("openshift", &openshiftMonitoring)
	return &globalMonitoring
```

Reply:
Good point. I've changed `central` code as well and renamed function to `getGlobalMonitoring` so `globalMonitoring` will not be shadowed